packages/devtools_app/lib/src/memory/memory_allocation_table_data.dart

```diff
@@ -9,6 +9,28 @@ import 'memory_protocol.dart';
 
 const defaultNumberFieldWidth = 100.0;
 
+class FieldTrace extends ColumnData<ClassHeapDetailStats> {
```

Comment:
lets use a real checkbox and make these clickable.

Reply:
Done.